example/companion_configuration/companion_configuration.cpp

```diff
@@ -0,0 +1,160 @@
+//
+// Simple example to demonstrate configuring MAVSDK for use
+// on a companion computer. This example simply streams telemetry.
```

Comment:
That's not true, it also tries to arm and all, right? :smile:

Reply:
@julianoes whoops - anyway thats fixed now